CHANGELOG.rst

```diff
@@ -17,6 +17,7 @@ This document describes changes between each past release.
   ``*_pool_recycle``, ``*_pool_timeout`` to control connections pool
   behaviour.
 - Follow redirections in batch subrequests (fixes #511)
+- Add a ``read_only`` setting to run the service in read-only mode. (#525)
```

Comment:
nit: `readonly` works well in one word too. Don't we to add some suffix, like `mode` or `resources` ?